### DISCUSSION TITLE
Why does my timeline only show 2008?

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3139] [Q&A]
![image](https://github.com/immich-app/immich/assets/7412219/78088dca-fc94-4cdf-a567-7a617fd4a449)

I have thousands of photos from 1999 to 2023, but the only year that ever shows on the right side is 2008.  I suspect the first photos I may have uploaded were from 2008 (Good chance I started with my wedding photos).

If I hover over the bar, I get whatever dates are appropriate, but that 2008 year sticks there.

thoughts?

**Comment by bcsteeve** [comment]
I also noticed it isn't even accurately placed lol.  See this screenshot of when I hover over or even below "2008":

![image](https://github.com/immich-app/immich/assets/7412219/aa4c1aab-e27e-4f34-97e4-95ff97e2ecb5)

**Comment by aureateflux** [comment]
Usually problems like this are caused by missing or bad metadata. It happens a lot when you manually download and back up images from Google or Apple ecosystems. There are tools available that will fix the metadata.

**Reply by bcsteeve** [reply]
Ok, fair point.  But in my case, I spent weeks making sure my entire collection was reasonably geocoded and time-stamped.  I after posting above, I added thousands of photos from a single day in 2019, and now I have two years showing... 2008 and 2019.

It seems like if there's a huge number of pics on a single date, that perhaps that year gets shown like this?  Strange if that's deliberate.

**Reply by alextran1502** [reply]
@bcsteeve how do you modify the date, what is the exif key that you use for your date time? We use exif tool to read the file metadata, so if you modify/add the date that is not conformed with exif, they might not get parse correctly

**Reply by bcsteeve** [reply]
I use exiftool to set the date created.  Immich seems to be perfectly aware of the dates of my images, it is just the way 2008 is being displayed as thought it is special that's weird.
